thirdparty/kyaml/runfn/runfn.go

```diff
@@ -455,15 +404,48 @@ func getUIDGID(asCurrentUser bool, currentUser currentUserFunc) (string, error)
 	return fmt.Sprintf("%s:%s", u.Uid, u.Gid), nil
 }
 
-// ffp provides function filters
-func (r *RunFns) ffp(spec runtimeutil.FunctionSpec, api *yaml.RNode, currentUser currentUserFunc) (kio.Filter, error) {
+// getFunctionConfig will return the config that used in 'functionConfig' field in
+// function input. If r.FnConfigPath is provided, it will read and return it. Otherwise
+// returns an error.
+func (r *RunFns) getFunctionConfig() (*yaml.RNode, error) {
```

Comment:
nit: small fixing in the comment (no need to describe the implementation logic):
```
getFunctionConfig returns yaml representation of functionConfig that can be provided to a function as input. 
```

Reply:
updated.